<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStatusBar" name="statusbar_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>3</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="centralwidget_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>546</width>
      <height>302</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
     <item>
      <layout class="QVBoxLayout" name="leftLayout">
       <item>
        <widget class="QPushButton" name="btnCreateSecond">
         <property name="text">
          <string>Создать второй</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnFinalizeSecond">
         <property name="text">
          <string>Завершить второй</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCompute">
         <property name="text">
          <string>Выполнить операцию</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnClear">
         <property name="text">
          <string>Очистить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbIntersect">
         <property name="text">
          <string>Пересечение</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbUnion">
         <property name="text">
          <string>Объединение</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbDifference">
         <property name="text">
          <string>Разность</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="canvasPlaceholder" native="true">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>300</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QStatusBar" name="statusbar_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>3</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
